test/unit/architecture_test.rb

```diff
@@ -17,8 +17,6 @@
     context "parameters" do
       it_should_accept "no arguments"
       it_should_accept_search_params
-      it_should_fail_with 'organization param', ['--organization-id=1']
-      it_should_fail_with 'location param', ['--location-id=1']
```

Comment:
Actually we could leave these tests as well, but test for `it_should_accept` instead :)

Reply:
Good idea I will do that.